packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -0,0 +1,685 @@
+        ) +
+          resolveErrorMessages([
+            `Consider adding a descriptive` +
+              chalk.hex('#e00000')(' 5 ') +
+              'to' +
+              chalk.hex('#e00000')(' 25 ') +
+              'word docblock in ' +
+              chalk.hex('#e00000')(fileName),
+          ]),
+        receivedErrorMessage(error),
+      );
+    }
+
+    // Message Description: Handles scenario where a docblock is received but doesn't meet the min and max requirements.
+    //
+    // It appears that "displayName" doesn't have a docblock between 5 and 25 words.
+    // It has a word count of: "docBlockLength"
+    // Here is "displayName"'s docblock: "docblock.description"
+    // To Resolve this issue:
+    // 1. Make sure that your docblock meets the required word count of 5 to 25 words.
+    else {
+      console.log(
+        defaultErrorMessage(
+          `has-docblock`,
+          displayName,
+          'docblock between 5 and 25 words. It has a word count of: ' +
+            chalk.green.italic(docBlockLength) +
+            paragraph() +
+            'Here is ' +
+            chalk.white(displayName) +
+            chalk.white(`'s`) +
+            ' docblock: ' +
+            paragraph() +
+            chalk.green.italic(docblock.description),
+        ) +
+          resolveErrorMessages([
+            `Make sure that your docblock meets the required word count of ` +
+              chalk.hex('#e00000')('5') +
+              ' to ' +
+              chalk.hex('#e00000')(`25`) +
+              ' words.',
+          ]),
+        receivedErrorMessage(error),
+      );
+    }
+    failedTests.push('has-docblock');
+  },
+
+  'exports-component': (componentInfo: ComponentDoc, testInfo: IsConformantOptions, error: string) => {
+    const { componentPath, displayName } = testInfo;
+    const fileName = path.basename(componentPath);
+
+    // Message Description: There are no matching exported components that match the displayName.
+    //
+    // It appears that "Component" doesn't have a export in: "componentPath"
+    // Here are the available exported component's from your file: "componentFile"
+    // To Resolve this issue:
+    // 1. Make sure that your component's "fileName" file contains an export for "displayName".
+    // 2. Check to see if you component uses export default and consider enabling useDefaultExport in your
+    // isConformant test.
+    console.log(
+      defaultErrorMessage(
+        `exports-component`,
+        displayName,
+        'export in: ' + paragraph() + chalk.green.italic(componentPath),
+      ) +
+        resolveErrorMessages([
+          `Make sure that your component's ` +
+            chalk.hex('#e00000')(fileName) +
+            ' file contains an export for ' +
+            chalk.hex('#e00000')(displayName) +
+            '.',
+          'Check to see if you component uses export default and consider enabling' +
```

Comment:
Would be good to either use backticks or a color here.
```suggestion
          'Check to see if you component uses `export default` and consider enabling' +
```